app/javascript/mastodon/components/extended_video_player.js

```diff
@@ -31,6 +32,12 @@ export default class ExtendedVideoPlayer extends React.PureComponent {
     this.video = c;
   }
 
+  handleClick = ev => {
```

Comment:
`ev` to represent event is not used in Mastodon. `e` or `event` is more common and would be preferred.
Anyway that is a trivial difference in style and can be ignored.